### DISCUSSION TITLE
How to create a voxelized source shape

### DISCUSSION THREAD

**Discussion by kamtchou98** [nrc-cnrc/EGSnrc#990] [Questions]
I have a phantom for which I sucessfully created and imported a mesh into egs. The only problem is that the phantom gets filled with Tc99m. Therefore I'm trying to create a source shape that would represent the empty space in the phantom. I have a mesh(.stl) file of that empty space and have managed to voxelate the .stl file. I found that there is an application egs_voxelized that could basically do this but I have no idea how to write the needed input file in binary. Is there an example file I could follow or even better is there an easier way for my source shape to be that empty space in my phantom? I've attached an image of the volume my source would need to be. Any help is much appreciated.
![Screenshot 2023-04-24 174002](https://user-images.githubusercontent.com/60777335/234136937-6e8c1c1c-8ced-4270-866d-53c65e894be0.png)

**Comment by rtownson** [comment]
I'd just like to note that as far as I know this has been resolved by email. They followed the format of the binary file in https://nrc-cnrc.github.io/EGSnrc/doc/pirs898/classEGS__VoxelizedShape.html to create a distributed radiation source.
